meerkat_libs/db_adapters/postgresql.py

```diff
@@ -0,0 +1,246 @@
+import psycopg2
+import logging
+import json
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+from psycopg2 import sql
+from psycopg2.extras import Json
+
+
+class PostgreSQLAdapter():
+
+    def __init__(self, db_name, structure, user='postgres', host='db'):
+        self.db_name = db_name
+        self.structure = structure
+        self.user = user
+        self.host = host
+        self.keys = self._extract_keys()
```

Comment:
As in DynamoDB adapter I suggest moving the call to `self._extract_keys` to the `setup()` method.

Reply:
See my above comment